sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -663,7 +661,7 @@ class Analyzer(
    *
    * [[ResolveRelations]] still resolves v1 tables.
    */
-  object ResolveTables extends Rule[LogicalPlan] {
+  object ResolveV2Relations extends Rule[LogicalPlan] {
```

Comment:
Why was this renamed to `ResolveV2Relations`? It doesn't handle views, so the name should stay `ResolveTables`.

Reply:
This rule only resolves `UnresolvedRelation` to `DataSourceV2Relation` so I think the new name fine.

I need this renaming because I added a new rule `ResolveTables` which resolves the identifier to a `Table` instance. Do you have a better idea about how to naming these rules?